graphics/rawtherapee/files/patch-libjpeg-turbo.diff

```diff
@@ -0,0 +1,18 @@
+--- rtengine/jdatasrc.cc	2024-02-16 13:12:09.000000000 +0800
++++ rtengine/jdatasrc.cc	2024-05-14 06:40:36.000000000 +0800
+@@ -252,14 +252,12 @@
+ #define JVERSION	"6b  27-Mar-1998"
+ #define JCOPYRIGHT_SHORT	"(C) 1998, Thomas G. Lane"
+ #define JMESSAGE(code,string)	string ,
++#endif
+ 
+ const char * const jpeg_std_message_table[] = {
```

Comment:
Because this changes the installed files (it is now providing an implementation of `jpeg_std_message_table` whereas before it was using an external implementation provided by libjpeg-turbo), the `revision` must be increased.

Reply:
Indeed. Will do soon once back to the laptop.

Reply:
@ryandesign Done